cli/internal/helm/update-aws-load-balancer-chart.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# updates the Helm charts for the AWS Load Balancer Controller in the CLI.
+# script is mostly copied from cli/internal/helm/update-csi-charts.sh
+
+set -euo pipefail
+set -o errtrace
+shopt -s inherit_errexit
+
+echo "Updating AWS Load Balancer Controller Helm chart..."
+branch="v0.0.140" # releases can update the AWS load-balancer-controller chart
+# Required tools
+if ! command -v git &> /dev/null; then
+  echo "git could not be found"
+  exit 1
+fi
+
+callDir=$(pwd)
+repo_tmp_dir=$(mktemp -d)
+
+chart_base_path="charts"
+chart_name="aws-load-balancer-controller"
+
+chart_url="https://github.com/aws/eks-charts"
+chart_dir="stable/aws-load-balancer-controller"
+cd "${repo_tmp_dir}"
+git clone \
+  --filter=blob:none \
+  --no-checkout \
+  --sparse \
+  --depth 1 \
+  --branch="${branch}" \
+  "${chart_url}" "${repo_tmp_dir}"
+
+git sparse-checkout add "${chart_dir}"
+git checkout
+cd "${callDir}"
+
+# remove values.yaml from upstream chart
+rm "${repo_tmp_dir}/${chart_dir}/values.yaml"
```

Comment:
Why do we remove the upstream values file and keep our own copy?
I think this could result in some annoying issues if the upstream chart introduces changes to the values file and the deployment templates.

Is there any particular reason for doing this, instead of "dynamically" updating values similar to how we do it for other charts, e.g. Cilium or cert-manager?

Reply:
https://github.com/edgelesssys/constellation/pull/2090#discussion_r1266948181 it makes it easier to test with `helm install` by keeping static values in the yaml and only injecting dynamic values.
On the downside it could cause issues with chart updates. Can inject dynamically if better

Reply:
While I agree with the comment by @derpsteb , the AWS loadbalancer chart is not controlled by us, so we shouldn't add to the values.yaml directly.